internal/plugin.go

```diff
@@ -598,10 +609,19 @@ func (p *Plugin) assertValidConfig() error {
 			"policyDefaults must provide only one of placement.labelSelector or placement.clusterSelectors",
 		)
 	}
-	if (len(p.PolicyDefaults.Placement.LabelSelector) != 0 || len(p.PolicyDefaults.Placement.ClusterSelectors) != 0) &&
-		(p.PolicyDefaults.Placement.PlacementRulePath != "" || p.PolicyDefaults.Placement.PlacementPath != "") {
+	if p.PolicyDefaults.Placement.PlacementRuleName != "" && p.PolicyDefaults.Placement.PlacementName != "" {
+		return errors.New(
+			"policyDefaults must provide only one of placement.placementName or placement.placementRuleName",
+		)
+	}
+
+	isDefaultSelector := (len(p.PolicyDefaults.Placement.LabelSelector) != 0 || len(p.PolicyDefaults.Placement.ClusterSelectors) != 0)
+	isDefaultPath := (p.PolicyDefaults.Placement.PlacementRulePath != "" || p.PolicyDefaults.Placement.PlacementPath != "")
+	isDefaultName := (p.PolicyDefaults.Placement.PlacementRuleName != "" || p.PolicyDefaults.Placement.PlacementName != "")
+	// if  label/cluster selector AND placement/rule path OR placement/rule name are defined OR if placement/rule path and name are defined
+	if (isDefaultSelector && (isDefaultPath || isDefaultName)) || (isDefaultPath && isDefaultName) {
```

Comment:
I think this would be easier to read:
```suggestion
	defaultPlacementOptions := 0
	if len(p.PolicyDefaults.Placement.LabelSelector) != 0 || len(p.PolicyDefaults.Placement.ClusterSelectors) != 0 {
		defaultPlacementOptions += 1
	}
	if p.PolicyDefaults.Placement.PlacementRulePath != "" || p.PolicyDefaults.Placement.PlacementPath != "" {
		defaultPlacementOptions += 1
	}
	if p.PolicyDefaults.Placement.PlacementRuleName != "" || p.PolicyDefaults.Placement.PlacementName != "" {
		defaultPlacementOptions += 1
	}

	if defaultPlacementOptions > 1 {
```